security/firewall.md

```diff
@@ -204,6 +206,10 @@ the service
 
 > Note: If you want to expose the service on multiple interfaces, repeat the
   steps described in part 1 for each interface
+  
+> Note: On Qubes R4, nftables is also used which imply that nft rules also need to be set. Qubes OS has defined a `qubes-firewall` table with a forward chain.
+
+`nft add rule ip qubes-firewall forward meta iifname eth0 ip daddr 10.137.0.x tcp dport 443 ct state new counter accept`
```

Comment:
Hmm, qubes-firewall service (by design) should only take care about traffic outgoing from a VM, not incoming. So, I'd consider the need for this line a bug. Let me fix it right now. And then replace this line with reference for appropriate package version.

Reply:
@marmarek, I suspect you're very busy. This line is to document this exact use case (expose service in AppVM and open flow inbound through sys-net then sys-firewall) The sub-section title is something like expose a service to the outside world. I took care to lock it as much as possible at every stage.

Reply:
The line as it is looks ok until the bug is resolved. Please add this ticket reference: https://github.com/QubesOS/qubes-issues/issues/3644